plugin/kubernetes/controller.go

```diff
@@ -41,9 +42,14 @@ type dnsController interface {
 	Run()
 	HasSynced() bool
 	Stop() error
+
+	// Modified returns last modified.
```

Comment:
Think this is better phrases as: `Modified returns the timestamp of the most recent changes`